testing/testcafe/tests/dataGrid/columnChooser.ts

```diff
@@ -42,3 +42,73 @@ test('Column chooser screenshot', async (t) => {
     enabled: true,
   },
 }));
+
+test('Column chooser checkboxes should be aligned correctly with plain structure', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  const dataGrid = new DataGrid('#container');
+
+  await t
+    .click(dataGrid.getHeaderPanel().getColumnChooserButton())
+    .wait(500);
```

Comment:
Why do we add ```wait``` here?

Reply:
I added this because of the CLICK_TIMEOUT, but you are right `wait` is redundant here, because I compare screenshot only of the column chooser and not the whole grid.